cypress/plugins/index.ts

```diff
@@ -0,0 +1,88 @@
+/// <reference types="cypress" />
+// ***********************************************************
+// This example plugins/index.js can be used to load plugins
+//
+// You can change the location of this file or turn off loading
+// the plugins file with the 'pluginsFile' configuration option.
+//
+// You can read more here:
+// https://on.cypress.io/plugins-guide
+// ***********************************************************
+
+// This function is called when a project is opened or re-opened (e.g. due to
+// the project's config changing)
+
+import { Collection, Db, MongoClient } from 'mongodb';
+import { SingletonDefinition, Team, User, ApplicationStatus, RSVPStatus } from '../../common/types';
+import { CreateUserInBackendArg } from '../types';
+
+type NextAuthVerificationToken = {
+  identifier: string;
+  token: string;
+  expires: Date;
+};
+type CypressMongoCtx = {
+  client: MongoClient;
+  serverDb: {
+    db: Db;
+    userDataCollection: Collection<User>;
+    singletonDataCollection: Collection<SingletonDefinition>;
+    teamDataCollection: Collection<Team>;
+  };
+  nextAuthDb: {
+    db: Db;
+    verificationTokens: Collection<NextAuthVerificationToken>;
+  };
+};
+
+// can't use normal connectToDatabase because we don't have access to env vars
```

Comment:
open an issue to refactor these and make the `MongoCtx` and `CypressMongoCtx` types the same, as well as re-use functionality

Reply:
see #109